lib/matplotlib/animation.py

```diff
@@ -554,7 +554,38 @@ def isAvailable(cls):
             return False
         return True
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, loop=0, **kwargs):
+        '''PillowWriter
+
+        Parameters
+        ----------
+        fps: int
+            Framerate for the gif.
+        loop: int
+            The number of times that the gif will loop.
+            A value of 0 is endless.
+        codec: string or None, optional
```

Comment:
The rest of the options are just ignored in the pillow writer so I don't think that they should be documented.